helm/kanister-operator/values.yaml

```diff
@@ -89,3 +89,22 @@ tolerations: []
 #
 # node labels for pod assignment. Evaluated as template
 nodeSelector: {}
+
+# Pod-level security context settings
+podSecurityContext:
+  runAsUser: 1000
+  fsGroup: 2000
+  runAsNonRoot: true
+  # SELinux options can be specified here if required
+  # seLinuxOptions:
+  #   level: "s0:c123,c456"
+
+# Container-level security context settings
+containerSecurityContext:
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  # AllowPrivilegeEscalation should be disabled for security
+  allowPrivilegeEscalation: false
```

Comment:
```suggestion
  allowPrivilegeEscalation: false
  
```

Reply:
@pavannd1 suggested changes are same as before. Should i change it to true ?

Reply:
@anishbista60 the change adds a new line at the end, before the file did not end in EOL.